utils/scripts/ocm/README.md

```diff
@@ -0,0 +1,119 @@
+# Openshift Dedicated Cluster operations in AWS with OCM CLI and Openshift API
+Script to provision Openshift Dediated (OSD) on AWS and with other functionalities like creating identity providers(IDP) to access the cluster, installing/uninstalling add-ons and retreiving the cluster info with the aid of Openshift Cluster Manager CLI tool ([ocm-cli](https://github.com/openshift-online/ocm-cli.git))
+
+# Prerequisites
+- Python 3.x
+- [Install ocm cli](#Install%20OCM-CLI)
+- access_key and secret_access_key for the AWS account user *ocdCcsAdmin*
```

Comment:
What account is `ocdCcsAdmin`?

Reply:
Hi @lugi0, This is a aws user who has policy (_**OSDRequirement**_) defined to deploy OSD on AWS